content/docs/connect/choose-connection.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Choosing your connection type
+subtitle: Rules of thumb for choosing one connection type over another
+enableTableOfContents: true
+---
+
+When setting up your application’s connection to your Neon Postgres database, you need to make two main choices: the type of driver and the type of connection. This flowchart will guide you through these options.
+
+## Choosing your connection type: flowchart
+
+![choose your connection type](/docs/connect/choose_connection.png)
+
+## Choosing your connection type: drivers and pooling
+
+### Step 1: Choose your Postgres driver
+
+Your first choice is which driver to use. If working in a serverless environment, use the [Neon Serverless Driver](/docs/serverless/serverless-driver). It's built to handle dynamic workloads with high variability in traffic &#8212; for example, Vercel Edge Functions or Cloudflare Workers.
```

Comment:
```suggestion
Your first choice is which driver to use. If working in a serverless environment and connecting from a JavaScript and TypeScript application, consider using the [Neon Serverless Driver](/docs/serverless/serverless-driver).  This is a low-latency Postgres driver that allows you to query data from serverless and edge environments over HTTP or WebSockets instead of TCP. It's built to handle dynamic workloads with high variability in traffic &#8212; for example, Vercel Edge Functions or Cloudflare Workers.
```